aems-edge/configurations/docker/Dockerfile-ilc

```diff
@@ -0,0 +1,30 @@
+ARG image_user=amd64
+ARG image_repo=eclipsevolttron/volttron-aems
+ARG image_tag=latest
+
+FROM ${image_repo}:${image_tag} AS volttron_aems
+
+SHELL [ "bash", "-c" ]
+
+USER $VOLTTRON_USER
+
+# copy over aems repo
+# May replace this with a git clone in the future
+COPY --chown=volttron:volttron volttron-ilc /code/ilc
+
+# Install libraries required for manager agent
+RUN python -m pip install sympy>=1.12
```

Comment:
Are these not in the base image requirements?  I am confused as to why these are necessary for the manager.

Reply:
I don't see sympy in the requirements.py file in volttron and there is no requirements file in either the aems or ilc repos.